src/transformers/utils/import_utils.py

```diff
@@ -522,6 +522,8 @@ def is_torch_bf16_gpu_available() -> bool:
         return torch.cuda.is_bf16_supported()
     if torch.xpu.is_available():
         return torch.xpu.is_bf16_supported()
+    if is_torch_hpu_available() and not is_habana_gaudi1():
```

Comment:
bf16 is available on on gaudi1
```suggestion
    if is_torch_hpu_available():
```